Core/src/core/utils/CommandUtils.ts

```diff
@@ -9,8 +9,31 @@ import {Guilds} from "../database/game/models/Guild";
 import {RequirementGuildNeededPacket} from "../../../../Lib/src/packets/commands/requirements/RequirementGuildNeededPacket";
 import {RequirementGuildRolePacket} from "../../../../Lib/src/packets/commands/requirements/RequirementGuildRolePacket";
 import {RequirementRightPacket} from "../../../../Lib/src/packets/commands/requirements/RequirementRightPacket";
+import {BlockingUtils} from "./BlockingUtils";
+import {draftBotInstance} from "../../index";
+
+type Requirements = {
+	disallowedEffects?: Effect[];
+	allowedEffects?: Effect[];
+	level?: number;
+	rightGroup?: RightGroup;
+	guildNeeded?: boolean;
+	guildRoleNeeded?: GuildRole;
+	notBlocked: boolean;
+};
 
 export abstract class CommandUtils {
+	static readonly DISALLOWED_EFFECTS = {
+		NOT_DEAD: [Effect.DEAD],
```

Comment:
Je ne comprends pas pourquoi tu as mis NOT partout (ou tu l'as enlevé). `disallowedEffects: CommandUtils.DISALLOWED_EFFECTS.NOT_DEAD` veut dire par exemple que tu n'autorises pas les gens qui ne sont pas mort, alors que tu dire l'inverse.

Reply:
`disallowedEffects: CommandUtils.DISALLOWED_EFFECTS.DEAD` ça veut dire que tu n'autorises pas les gens morts, c'est mieux